qdrant-landing/content/articles/fastembed.md

```diff
@@ -0,0 +1,238 @@
+---
+title: "FastEmbed: Fastest Embedding Generation for Text"
+short_description: "FastEmbed: Quantized Embedding models with ONNXRuntime"
+description: "FastEmbed is a Python library engineered for speed, efficiency, and accuracy"
+social_preview_image: /articles_data/fastembed/preview/social_preview.png
+preview_dir: /articles_data/fastembed/preview
```

Comment:
There needs to be an icon added. Please have a look at some other articles. Each of them should also have an SVG icon:

```markdown
small_preview_image: /articles_data/new-recommendation-api/icon.svg
```

Right now, it's not being displayed properly in the footer:

![image](https://github.com/qdrant/landing_page/assets/2649301/e9903b0e-447f-4193-a40f-7d2cd4f6d51a)

Reply:
How do I get that? Can use a lightning SVG perhaps. How do we get a SVG which is okay for commercial use?

Reply:
Some websites expose icons free to be used commercially, even without attribution. For example:
https://www.reshot.com/free-svg-icons/

Reply:
Added a SVG icon now